### ISSUE TITLE
страшный грех разработки

### ISSUE THREAD

**Issue by wldmr-key** [romankuz19/web-application#2]
Это - подобные вещи - 
$mysql=new mysqli('localhost','root','root','to-do');

коннекшн стринг и прочие подобные шштуки выносятся в конфиги, вообще констанст в хардкоде должно быть минимально необходимое количество

И ошибки нужно обрабатывать и в php тоже ))

**Comment by romankuz19** [comment]
Да, я закинул в конфиг, вроде повторения лишние ушли, обработчик ошибок у меня в check.php

**Comment by wldmr-key** [comment]
ок
